CHANGELOG.md

```diff
@@ -2,11 +2,20 @@
 
 Records breaking changes from major version bumps
 
+## 22.4.0
```

Comment:
Only required to update the changelog for breaking changes... But I'm not especially opposed to more documentation. :)